2 Niagara Fundamentals Exercise - Multiplexer/multiplexer-rt/module.palette

```diff
@@ -1,5 +1,165 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <bajaObjectGraph version="4.0" reversibleEncodingKeySource="none" FIPSEnabled="false" reversibleEncodingValidator="[null.1]=" >
 <p m="b=baja" t="b:Folder">
+<p m="mux=multiplexer" t="mux:Multiplexer" n="Multiplexer" />
```

Comment:
This is the only line where you need `m="mux=multiplexer"` in. Once you've declared a symbol for your module name once, you don't need to redeclare it.